draft-amjad-cfrg-partially-blind-rsa.md

```diff
@@ -166,10 +166,10 @@ they satisfy a particular criteria. In particular, each RSA modulus for a key pa
 MUST be the product of two safe primes p and q. A safe prime p is a prime number
 such that p = 2q + 1, where q is also a prime number.
 
-A signing key pair is a tuple (skS, pkS), where each element is as follows:
+A signing key pair is a tuple (sk, pk), where each element is as follows:
 
-- skS = (p, q, phi, d), where phi = (p - 1)(q - 1)
-- pkS = (n, e), where n = p * q and d * e == 1 mod phi.
+- sk = (p, q, phi, d), where phi = (p - 1)(q - 1)
+- pk = (n, e), where n = p * q and d * e == 1 mod phi.
```

Comment:
Can remove e and d as follows:

sk = (p, q, phi)
pk = n

Reply:
Why would we remove `e` and `d`?

Reply:
Because e & d are not used anywhere anymore. Before we would use e & d in the augment functions. However, in the new Derive* functions, they are not needed. The public key one would only need modulus n whereas the secret key one would need p, q, phi.

Reply:
Let's address this in a separate change.